Thing.h

```diff
@@ -715,7 +713,8 @@ class ThingDevice {
         descr.createNestedObject("securityDefinitions");
     JsonObject nosecSc = securityDefinitions.createNestedObject("nosec_sc");
     nosecSc["scheme"] = "nosec";
-    descr["security"] = "nosec_sc";
+    JsonArray securityJson = descr.createNestedArray("security");
+    securityJson.add("nosec_sc");
```

Comment:
What's the reason for making this an array? A string should also be valid (see https://w3c.github.io/wot-thing-description/#thing).

Reply:
Well, then I don't need to worry, if it is one of multiple security schema.

Reply:
I don't understand your response. It seems this is another unnecessary change?

Reply:
It makes it easier to add another security schema. But I think I did that because of some compatibility issues. I am not sure anymore.